openwisp-qa-check

```diff
@@ -96,15 +95,15 @@ runblack() {
 # 1. --force-checkcommit is specified
 # 2. This function is invoked from Travis building a Pull Request
 runcheckcommit() {
-  [ -n "$TRAVIS_PULL_REQUEST" ] || TRAVIS_PULL_REQUEST=false
```

Comment:
I didn't investigate why this was not running, instead I made it run on local machine & travis both! :smile: